mimic/test/test_auth.py

```diff
@@ -1286,3 +1286,75 @@ def test_string_errors_as_well_as_json_errors(self):
             self, root, username="failme", request_func=request_with_content)
         self.assertEqual(response.code, 500)
         self.assertEqual(body, "Failure of JSON")
+
+
+class IdentityNondedicatedFixtureTests(SynchronousTestCase):
+
+    def test_non_dedicated_tokens(self):
+        """
+        Obtain Identity entries when presented tokens issued to non-dedicated users
+        """
+        url = "/identity/v2.0/tokens"
+        core, root = core_and_root([])
+        (response, content) = self.successResultOf(
+            json_request(self, root, "GET", url + "/OneTwo"))
+        self.assertEqual(200, response.code)
+        self.assertTrue(content["access"])
+        self.assertTrue(content["access"]["token"])
+        self.assertTrue(content["access"]["token"]["tenant"])
+        self.assertTrue(content["access"]["token"]["tenant"]["id"])
+        self.assertEqual(content["access"]["token"]["tenant"]["id"], "135790")
+
+        (response, content) = self.successResultOf(
+            json_request(self, root, "GET", url + "/ThreeFour"))
+        self.assertEqual(200, response.code)
+        self.assertTrue(content["access"])
+
+
+class IdentityDedicatedFixtureTests(SynchronousTestCase):
+
+    def test_dedicated_tokens(self):
+        """
+        Obtain Identity entries when presented tokens issued to dedicated users
+        """
+        url = "/identity/v2.0/tokens"
+        core, root = core_and_root([])
+        (response, content) = self.successResultOf(
+            json_request(self, root, "GET", url + "/HybridOneTwo"))
+        self.assertEqual(200, response.code)
+        self.assertTrue(content["access"])
+        self.assertTrue(content["access"]["token"])
+        self.assertTrue(content["access"]["token"]["tenant"])
+        self.assertTrue(content["access"]["token"]["tenant"]["id"])
+        self.assertEqual(content["access"]["token"]["tenant"]["id"], "hybrid:123456")
```

Comment:
IMO all the assertions leading up to this one are redundant, since this will throw an exception if the response is malformed.